RecoParticleFlow/PFClusterProducer/src/GenericSimClusterMapper.h

```diff
@@ -0,0 +1,38 @@
+#ifndef __GenericSimClusterMapper_H__
+#define __GenericSimClusterMapper_H__
+
+#include "RecoParticleFlow/PFClusterProducer/interface/InitialClusteringStepBase.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecHitFraction.h"
+
+#include "SimDataFormats/CaloAnalysis/interface/SimClusterFwd.h"
```

Comment:
I think due to this dependence we better move GenericSimClusterMapper to another package.
Say, RecoParticleFlow/PFSimClusterProducer
@davidlange6 @Dr15Jones 
please comment.